app/Console/Commands/OrderProducts.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Exports\OrdersExport;
+use App\Models\Order;
+use App\Models\Product;
+use Faker\Factory;
+use Illuminate\Console\Command;
+use Maatwebsite\Excel\Facades\Excel;
+
+class OrderProducts extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'order:products';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Order products automatically based on their live count.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+
+    public function handle() :void
+    {
+        $faker = Factory::create();
+        $products = Product::all();
+
+        foreach ($products as $product)
```

Comment:
Reikėtų iškelti šitą logiką į atskirą klasę ir išextractinti į mažesnius metodus.
Labai sunku suprasti, kas konkrečiai yra daroma čia.